lib/flags.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+class Flags {
+  static from(...args) {
+    const values = new Map(args.map((v, i) => [v, i]));
+    class FlagsClass {
+      static from(...args) {
+        return new FlagsClass(...args);
+      }
+
+      constructor(...args) {
+        this.value = 0;
+        args.forEach(arg => this.set(arg));
+      }
+
+      has(key) {
+        return values.has(key);
+      }
+
+      get(key) {
+        return this.value >> values.get(key) & 0x1;
```

Comment:
Why are you not validating that `key` provided exists in the `values` Map as you do in other methods?